frontend/public/components/dashboard/inventory-card/inventory-item.tsx

```diff
@@ -69,7 +69,7 @@ export const InventoryItem: React.FC<InventoryItemProps> = ({ isLoading, singula
   );
 };
 
-const Status: React.FC<StatusProps> = React.memo(({ groupID, count, statusIDs, kind, namespace, filterType}) => {
+const LinkStatus: React.FC<LinkStatusProps> = React.memo(({ groupID, count, statusIDs, kind, namespace, filterType}) => {
```

Comment:
nit: I would call this `StatusLink`. The other way sounds like it's the status of the link.

Reply:
Agreed, done